CONTRIBUTING.md

```diff
@@ -74,9 +74,11 @@ Run a single test
 `$ rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"`
 
 Run tests against different Rails versions by setting the RAILS_VERSION variable
-and bundling gems.
+and bundling gems.  (save this script somewhere executable and run from top of AMS repository)
```

Comment:
Dont know much about WWTD .... I was originally gonna run it in the console, but found it simpler just to ``../ams_tester.sh`` and bail out when something crapped out 😩